src/main/java/core/basesyntax/RightTriangle.java

```diff
@@ -0,0 +1,26 @@
+package core.basesyntax;
+
+public class RightTriangle extends Figure {
+    private final int side1;
+    private final int side2;
```

Comment:
Use meaningful names for variables. 
```suggestion
    private final int adjacentSide;
    private final int oppositeSide;
```